README.md

```diff
@@ -83,6 +83,11 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | `bbb_monitoring_all_in_one_htpasswd` | Deprecated, use `bbb_monitoring_htpasswd` instead | | |
 | `bbb_monitoring_recordings_from_disk` | Collect recordings metrics by querying the disk instead of the API. See [this](https://bigbluebutton-exporter.greenstatic.dev/exporter-user-guide/#optimizations) for details. | `true` |
 | `bbb_monitoring_external` | Enable exposure to nginx | `false` | Can be reached under `/mon/bbb` and `/mon/node` - requires `htpasswd` and `htpasswd_user`. If `bbb_monitoring_systemd_enable` is enabled, no Node Exporter installation process is included |
+| `bbb_monitoring_external_metrics_port` | Port of bbb-exporter | `9688` |  default port 9866 is defined by the exporter [itself](https://github.com/greenstatic/bigbluebutton-exporter/blob/master/bbb-exporter/settings.py#L37) |
+| `bbb_monitoring_external_node_port` | Port of node-exporter | `9100` | default port 9100 is by the bbb-exporter [itself](https://github.com/greenstatic/bigbluebutton-exporter/blob/master/extras/node_exporter/docker-compose.yaml#L10) |
+| `bbb_monitoring_use_ip_restriction` | Restricting access by IP Address instead using htpasswd | `false` | |
+| `bbb_monitoring_use_restriction_combination`| Use combination of IP Address restriction and htpasswd | `false` | |
+| `bbb_monitoring_external_ip_list` | A list of ip addresses for restricting access | `[]` | in array with key `address` |
```

Comment:
I would suggest this variable to be renamed 'bbb_monitoring_external_ip_whitelist' for clarity. It's not unclear now, but the change would make it even better in my opinion.

Reply:
The variable has been renamed